spec/features/admin/user_orders/show_spec.rb

```diff
@@ -16,10 +16,9 @@
 
       allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(site_admin)
 
-      visit admin_user_show_path(user_1.id)
```

Comment:
Was this not working?

Reply:
We didn't  encounter any issues using this path helper in our testing.